test/WebSites/ActivatorWebSite/TagHelpers/FooterTagHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace ActivatorWebSite.TagHelpers
+{
+    [HtmlElementName("body")]
+    public class FooterTagHelper : TagHelper
+    {
+        [Activate]
+        public IHtmlHelper HtmlHelper { get; set; }
+
+        [Activate]
+        public ViewDataDictionary ViewData { get; set; }
+
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            var builder = new TagBuilder("footer");
```

Comment:
agree we don't need the `TagBuilder` here.  but to avoid demonstrating a poor practise, suggest instead `output.PostContent = $"<footer>{HtmlHelper.Encode(ViewData["footer"])}</footer>";`.  should never ignore HTML encoding.